src/main/kotlin/no/nav/familie/ef/sak/mapper/SivilstandMapper.kt

```diff
@@ -0,0 +1,31 @@
+package no.nav.familie.ef.sak.mapper
+
+import no.nav.familie.ef.sak.api.dto.SivilstandInngangsvilkårDto
+import no.nav.familie.ef.sak.api.dto.SivilstandRegistergrunnlagDto
+import no.nav.familie.ef.sak.api.dto.SivilstandSøknadsgrunnlagDto
+import no.nav.familie.ef.sak.api.dto.Sivilstandstype
+import no.nav.familie.ef.sak.integration.dto.pdl.PdlSøker
+import no.nav.familie.ef.sak.repository.domain.søknad.Sivilstand
+
+object SivilstandMapper {
+
+    fun tilDto(sivilstandsdetaljer: Sivilstand, pdlSøker: PdlSøker): SivilstandInngangsvilkårDto {
+        val sivilstand = pdlSøker.sivilstand.first()
```

Comment:
Er `first` alltid den nyeste sivilstanden i lista? :) Hvis vi ikke er helt sikre på det, går det kanskje an å gjøre en `maxBy { it.gyldigFraOgMed }` 😄

Reply:
Veldig god poeng Martine. Jeg har missoppfattet dette.
Er ikke helt sikker på att det er gyldigFraOgMed som vi skal bruke heller, må kanskje sjekke om FREG er master og sjekke att den ikke er historisk.. 

Så lurer på om vi kanskje kan ta det i en egen oppgave.

Jeg opprettet en oppgave for søknad etter din kommentar, https://favro.com/organization/98c34fb974ce445eac854de0/a64c6aad9b0d61ef6c0290bd?card=Tea-2977

Reply:
Kan hende det er jeg som er helt noob, men hva er vanlig å gjøre i kotlin hvis `maxBy`-funksjonen forventer `LocalDate` når `it.GyldigFraOgMed` er optional? @blommish @marenger 

```
data class Sivilstand(val type: Sivilstandstype,
                      val gyldigFraOgMed: LocalDate?,
```

Reply:
Vi må gå igjennom all bruk av PDL og dette med å hente ut den "siste", så vi fikser det i den favro-oppgaven som jeg linket til tidligere